localized/games/wordwindow/wordwindow_level_en.tsv

```diff
@@ -73,11 +73,11 @@ en-US	6	1	1	4	"Ganesh has 4 stamps. Daanish has 5 more stamps than Ganesh. How m
 en-US	6	1	2	4	"The library has 5 more tables than the school office. The school office has@2 tables. How many tables does the library have?"	The library has 5 more tables than the school office. The school office has@2 tables. How many tables does the library have?	5	7	9	11	tables	tables	tables	tables	2 + 5 = <7>	2	7	2	0	0					wordwindow-stone-green.png	wordwindow-stone-red.png		
 en-US	6	1	3	4	"The school has two lost-and-found boxes. In the first box, there are@5 hats. In the second box, there are@4 more hats than in the first box. How many hats are there in the second box?"	The school has two lost-and-found boxes. In the first box, there are@5 hats. In the second box, there are@4 more hats than in the first box. How many hats are there in the second box?	5	9	14	15	hats	hats	hats	hats	5 + 4 = <9>	2	5	9	0	0					wordwindow-stone-yellow.png	wordwindow-stone-red.png		
 en-US	6	1	4	4	"There are 10 plates at the dinner table. There are 5 more people than plates. How many people are at dinner?"	There are 10 plates at the dinner table. There are 5 more people than plates. How many people are at dinner?	10	15	20	25	people	people	people	people	10 + 5 = <15>	2	10	15	0	0					wordwindow-stone-blue.png	wordwindow-stone-red.png		
-en-US	6	1	5	4	"There are 14 chairs in Ms. Jeet's classroom. There are 3 more chairs in Mr. Nakul's classroom. How many chairs are there in Mr. Nakul's classroom?"	There are 14 chairs in Ms. Jeet's classroom. There are 3 more chairs in Mr. Nakul's classroom. How many chairs are there in Mr. Nakul's classroom?	11	14	17	20	chairs	chairs	chairs	chairs	14 + 3 = <17>	3	14	17	0	0					wordwindow-stone-blue.png	wordwindow-stone-yellow.png		
+en-US	6	1	5	4	"There are 14 chairs in Miss Jeet's classroom. There are 3 more chairs in Mr. Nakul's classroom. How many chairs are there in Mr. Nakul's classroom?"	There are 14 chairs in Miss Jeet's classroom. There are 3 more chairs in Mr. Nakul's classroom. How many chairs are there in Mr. Nakul's classroom?	11	14	17	20	chairs	chairs	chairs	chairs	14 + 3 = <17>	3	14	17	0	0					wordwindow-stone-blue.png	wordwindow-stone-yellow.png		
```

Comment:
Check if TTS works correctly for `Mr.`. Also, check if all the references of `Ms.` is replaced by `Miss`

Reply:
Checked Mr. is spoken correctly by the TTS, all the Ms. are replaced by Miss